docs/launch_boot_tests.py

```diff
@@ -112,6 +112,9 @@
                         run = gem5Run.createFSRun(
                             'gem5/build/X86/gem5.opt',
                             'configs-boot-tests/run_exit.py',
+                            '''results/X86/run_exit/vmlinux-5.2.3/
```

Comment:
Crazy.. this works?!?!

We should document somewhere that you're allowed to use triple quotes in any parameter of these functions.

Reply:
Sorry, I messed this up. It should not work (as expected) with tripe quotes (though it will run and generate an out dir on a path with a name containing undesired characters). I have pushed a new commit to fix this